docs/tutorials/connecting-to-the-internet-via-bluetooth.md

```diff
@@ -255,9 +255,54 @@ If you are using an older version, please upgrade.
 
         {% include /util/screenshot.html source="/images/brickman/networking-my-computer-0-disconnect-selected-online.png" %}
 
-*   {: tab="Windows"}__HELP WANTED!__ See [issue #287](https://github.com/ev3dev/ev3dev/issues/287).
+*   {: tab="Windows"}
```

Comment:
To be consistent, we should have an icon here. So, this line should be 

    {: tab="Windows"}{% include /style/icon.html type="info" %}